appcelerator/titanium_mobile
# [TIMOB-14742] Android: Fixed window/view "backgroundImage" assignment where it was loading the same image 4 times.

**JIRA:** https://jira.appcelerator.org/browse/TIMOB-14742

**Note:**
Titanium no longer prints each image it's loading to the Android log. So, QE is unable to verify that the image is loading just once or multiple times anymore. But please run the below test to verify that "backgroundImage" before still works for single image backgrounds and multi-state backgrounds.

**Testing:**

1. Build and run "KitchenSink" app for Android.
2. Tap "Button" item in the list.
3. Tap the "CLICK ME" button several times to verify it changes state like before.
4. Tap the "Back" button.
5. Tap the "Buttons States" item in the list.
6. Tap the "CLICK ME" button several times to verify it changes state like before.
